### PR TITLE
next: Forgot password reset password form handler

### PR DESCRIPTION
**Pull request by Steven-Chan**
connect #778

### CODE REVIEW

**Comment by Steven-Chan** [comment]
@carmenlau upadted

File: pkg/auth/handler/forgotpwd/reset.go [hunk 1]
```diff
@@ -33,6 +33,9 @@ func AttachForgotPasswordResetHandler(
 	server.Handle("/forgot_password/reset_password", &ForgotPasswordResetHandlerFactory{
 		authDependency,
 	}).Methods("OPTIONS", "POST")
+	server.Handle("/forgot_password/reset_password_form", &ForgotPasswordResetFormHandlerFactory{
+		authDependency,
+	}).Methods("POST", "GET")
```
Comment (kaojohnny) [inline pkg/auth/handler/forgotpwd/reset.go]: Do we need support "POST"?

Comment (Steven-Chan) [reply pkg/auth/handler/forgotpwd/reset.go]: Yes, https://github.com/SkygearIO/forgot_password/blob/master/forgot_password/handlers/reset_password.py#L238

**Comment by Steven-Chan** [comment]
@kaojohnny updated

**Pull request merged**